rocky/rocky/views/organization_member_list.py

```diff
@@ -25,15 +26,28 @@ class OrganizationMemberListView(
     ListView,
 ):
     model = OrganizationMember
-    filters_active: List[str] = []
     context_object_name = "members"
 
     def get_queryset(self):
-        return self.model.objects.filter(organization=self.organization)
+        queryset = self.model.objects.filter(organization=self.organization)
+        if "blocked_status_filter" in self.request.GET:
+            blocked_status_filter = self.request.GET.getlist("blocked_status_filter", [])
+            queryset = self.filter_queryset(queryset, blocked_status_filter)
+        for member in queryset:
+            if member.status == "blocked":
+                member.blocked = True
```

Comment:
Is this a dynamically added property now to use in the templates? Would be a good candidate for a computed property on the member class to resolve the hardcoded strings.

Reply:
Done